app/controllers/application_controller.rb

```diff
@@ -17,12 +17,13 @@ class ApplicationController < ActionController::Base
 
   layout :set_layout
 
-  rescue_from ApiEntity::Error, Errno::ECONNREFUSED do
+  rescue_from ApiEntity::Error, Faraday::ServerError, Errno::ECONNREFUSED do
```

Comment:
If we're intercepting these errors will Sentry receive them? I wonder if it's worth notifying sentry with the full error message?

Reply:
I think you can remove the reference to ApiEntity::Error, here?

Reply:
There are still some locations which are using ApiEntity::Error - it can be removed when those are.

Re: Sentry - yeh, I don't much like this but was following the existing behaviour - I'd be much happier removing the global rescue though